playwright/src/test/java/com/microsoft/playwright/TestBrowserContextBasic.java

```diff
@@ -16,20 +16,25 @@
 
 package com.microsoft.playwright;
 
-import org.junit.jupiter.api.Disabled;
+import com.microsoft.playwright.junit.UsePlaywright;
+import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
 
 import java.io.OutputStreamWriter;
 import java.util.ArrayList;
 import java.util.List;
 
+import static com.microsoft.playwright.Utils.isWebKit;
 import static com.microsoft.playwright.Utils.verifyViewport;
 import static java.util.Arrays.asList;
 import static org.junit.jupiter.api.Assertions.*;
 
-public class TestBrowserContextBasic extends TestBase {
+@Tag("fixtures") //temp tag to allow only tests with junit integration to run.  will be removed before merge.
+@UsePlaywright(browserFactory = BrowserFromEnv.class)
+public class TestBrowserContextBasic extends __TestBaseNew {
   @Test
-  void shouldCreateNewContext() {
+  void shouldCreateNewContext(Browser browser) {
+    //Failing: should be zero?
     assertEquals(1, browser.contexts().size());
```

Comment:
@yury-s this and another test below are failing for the same reason.  If using fixtures, i think these assertions should be updated to be 0 for the context list size.

Reply:
Yes, updating to 0 sounds right, see my response below.